tutorials/epochs/baseline_regression.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-baseline_regression:
+
+====================================================================
+Baseline correction using generalized linear mixed models
+====================================================================
+
+By including the baseline interval as a regressor in a generalized
+linear model, we allow the data to determine how much baseline
+correction is necessary for the specific epoch. This approach can
+increase statistical power (Alday et al., 2019).
+Alday et al. provided a script at osf: https://osf.io/85bjq
+"""
+# Authors: Carina Forster
```

Comment:
```suggestion
# Authors: Carina Forster
           (adapted from code by Philip Alday: https://osf.io/85bjq)
```
This is how I would do it, although maybe @palday would prefer to simply be listed as co-author on the tutorial?

Reply:
I don't have terribly strong preference either way, but it's `Phillip` (with two `l`'s) please 😄